### PR TITLE
Fix numerical collapse

### PR DESCRIPTION
**Pull request by rmfarrell**
Also logs error if wrong type of input is used.

### CODE REVIEW

**Comment by rmfarrell** [comment]
screen shot 2018-08-29 at 5 51 55 pm

File: inputs/text.vue [hunk 1]
```diff
@@ -64,11 +68,8 @@
     },
     computed: {
       type() {
-        if (this.args.type === 'multi-line' || !this.args.type) {
-          return 'text';
-        } else {
-          return this.args.type;
-        }
+        return  this.args.type === 'multi-line' || !this.args.type ?
```
Comment (nelsonpecora) [inline inputs/text.vue]: what is this line even

Comment (rmfarrell) [reply inputs/text.vue]: ah, I'm just turning this if into a ternary. eslint is stripping the `()` which I'd ordinarily keep in so it doesn't look confusing.

File: inputs/text.vue [hunk 2]
```diff
@@ -130,17 +135,20 @@
       },
       enableInput() {
         this.isDisabled = false;
-      }
+      },
```
Comment (nelsonpecora) [inline inputs/text.vue]: ,,,,,,,,,,

File: inputs/text.vue [hunk 3]
```diff
@@ -130,17 +135,20 @@
...
       if (this.initialFocus === this.name) {
-        const offset = _.get(this, '$store.state.ui.currentForm.initialOffset');
 
         this.$nextTick(() => {
-          if (_.includes(['text', 'search', 'url', 'tel', 'password', 'multi-line'], this.type)) {
-            // selection range is only permitted on text-like input types
-            setCaret(this.$el, offset, this.data);
+  
+          // validate input type
+          if (!validInputTypes.includes(this.type)) {
```
Comment (nelsonpecora) [inline inputs/text.vue]: good

**Pull request merged**